pkg/config/virtio.go

```diff
@@ -11,6 +11,27 @@ import (
 // The VirtioDevice interface is an interface which is implemented by all virtio devices.
 type VirtioDevice VMComponent
 
+const (
```

Comment:
maybe it makes sense to break this group of devices into something like graphics.go or display.go.  Dealer's choice.